src/api/cf.nim

```diff
@@ -26,12 +27,20 @@ const
   ## base url of the forgesvc endpoint
   modsBaseUrl = "https://addons-ecs.forgesvc.net/api/v2"
 
+
+converter getDependencies(json: JsonNode): int =
+  var id: int = json{"id"}.getInt()
+  if id == 0:
+    id = json{"addonId"}.getInt()
+  return id
```

Comment:
Mods can have multiple dependency types - Embedded Library, Optional Dependency, Required Dependency, Tool, Incompatible & Include.
The API returns a `type` together with the `addonId` - Since we only want required dependencies, we only want to accept dependencies with the type `3`. (You can check for yourself at https://addons-ecs.forgesvc.net/api/v2/addon/240630/file/2728585 = Just Enough Resources which has JEI as a required dependency)